WcaOnRails/app/models/concerns/resultable.rb

```diff
@@ -10,13 +10,17 @@ module Resultable
     # NOTE: We use cached values instead of belongs_to to improve performances.
     belongs_to :competition, foreign_key: :competitionId
     validates :competition, presence: true
+    alias_attribute :competition_id, :competitionId
```

Comment:
Side note: while I made this change so that it fits in the serializable options, it's worth noting that having this attribute aliased would let us get rid of the various `competitionId` (and its friends) used in some places of our codebase, to stick with rails naming convention.